starrocks/results/c6a.4xlarge.json

```diff
@@ -0,0 +1,55 @@
+{
+    "system": "StarRocks",
+    "date": "2022-07-15",
+    "machine": "c6a.4xlarge, 500gb gp2",
+    "cluster_size": 1,
+    "comment": "",
+    "tags": ["Vectorized", "Realtime", "StarRocks"],
```

Comment:
These tags are not used for other systems (Vectorized, Realtime) and StarRocks tag is pointless as it is only for this system. I will remove them.

Reply:
Also added "ClickHouse derivative" tag because of Aggregator, Volnitsky, RadixSort, UTF8, and some other chunks of derived code. Although the system is not a ClickHouse spin-off, just reusing some code and ideas.

Reply:
Sure, thanks.